src/DynamoCoreWpf/Views/SplashScreen/SplashScreen.xaml.cs

```diff
@@ -484,8 +488,12 @@ internal void CloseWindow()
             // RequestUpdateLoadBarStatus event needs to be unsubscribed when the splash screen window is closed, to avoid populating the info on splash screen.
             else
             {
-                this.Close();
-                viewModel?.Model.ShutDown(false);
+                Close();
+                viewModel.RequestClose -= SplashScreenRequestClose;
+                //viewModel?.Model.ShutDown(false);
+                //TODO ensure viewmodel is shutdown by view being disposed.
```

Comment:
I believe I verified this, but I will double check - IMO there is no reason to shutdown the view model because what we really need to do is dispose the DynamoView which will shutdown the view model - but again - I will verify.

Reply:
@QilongTang I've verified that the close call on DynamoView will shutdown the model and view model under normal circumstances. When in testmode this is handled by the teardown logic of most test base classes. I think this change is safe.
PTAL.